package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "elementor",
     "slug": "elementor",
-    "version": "3.7.8",
+    "version": "3.9.0",
```

Comment:
It will be updated automatically: 
https://github.com/elementor/elementor/commit/2d655000a648b1b9bf56f75a995f679a1d9cd9ae#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R4